build/squire-raw.js

```diff
@@ -430,6 +430,10 @@ function fixCursor ( node, root ) {
         return originalNode;
     }
 
+    if ( node.className && /protonmail_signature/.test(node.className) ) {
```

Comment:
```suggestion
    if ( /protonmail_signature/.test( node.className ) ) {
```

`/foo/.test(({}).bar)` is false so you don't need to test node.className